README.md

```diff
@@ -367,31 +370,62 @@ buildscript {
 apply plugin: 'com.squareup.sqldelight'
 ```
 
-You can declare databases explicitly in gradle for tighter control of the generated schema:
+On Android, the plugin will create a default database called `Database` using the project package name. For greater customization, you can declare databases explicitly:
 
+#### build.gradle
 ```groovy
 sqldelight {
-  MyDatabase {
-    packageName = "com.example.db"
-    // By default this is ["sqldelight"], and means your sqldelight will be in
-    // folders like 'src/main/db' instead of 'src/main/sqldelight'
-    sourceFolders = ["db"]
+    MyDatabase {
+        /**
+         * package name used for the generated MyDatabase.kt
+         */
+        packageName = "com.example.db"
+
+        /**
+         * An array of folders where the plugin will read your '.sq' and '.sqm' files.
+         * The folders are relative to the existing source set so if you specify ["db"],
+         * the plugin will look into 'src/main/db'
+         * Defaults to ["sqldelight"] (src/main/sqldelight)
+         */
+        sourceFolders = ["db"]
+
+        /**
+         * The directory where to store '.db' schema files relative to the root of the project.
+         * These files are used to verify that migrations yield a database with the latest schema.
+         *
+         * Defaults to null so the verification tasks will not be created.
+         */
+        schemaOutputDirectory = file("src/main/sqldelight/databases")
+
+        /**
+         * Optionally specify schema dependencies on other gradle projects
+         */
+        dependency project(':OtherProject')
+    }
+
+    // For native targets, chose wether sqlite should be automatically linked.
+    // Defaults to true.
+    linkSqlite = false
+}
+```
 
-    // Defaults to file("src/main/sqldelight")
-    schemaOutputDirectory = file("build/dbs")
+#### build.gradle.kts
```

Comment:
Move this below the next sentence and change to

```
`build.gradle.ktx`:
```
similar to what I suggested above. These shouldn't be headers.